src/components/menu/FileExplorerActionMenu.vue

```diff
@@ -147,16 +147,23 @@ function deleteFile() {
  * @param {object} file - Selected file.
  * @returns {Promise} Promise with nothing on success otherwise an error.
  */
-function addFile(file) {
+async function addFile(file) {
   loading.value.add = true;
   return gitAdd(props.projectName, file.id)
-    .then(() => {
+    .then(async () => {
       Notify.create({
         type: 'positive',
         message: t('actions.fileExplorer.file.add'),
         html: true,
       });
-      GitEvent.AddEvent.next(file.id);
+
+      const [fileStatus] = await getStatus(
+        props.projectName,
+        [file.id],
+        (path) => path === file.id,
+      );
+
+      GitEvent.AddEvent.next(fileStatus);
```

Comment:
Update doc for `AddEvent` inside `GitEvent.js`

Reply:
Done